BrainPortal/app/controllers/signups_controller.rb

```diff
@@ -266,6 +283,38 @@ def resend_conf_multi #:nodoc:
     render :action => :multi_action
   end
 
+  def hide_multi #:nodoc:
+    reqids = params[:reqids] || []
+    reqs   = Signup.find(reqids)
+
+    count = 0
```

Comment:
why a `count` variable that you increment once, when you can just use `reqs.size` ?!? It's only used for the `pluralize` message after all.